tests/CommonTests.cs

```diff
@@ -121,6 +121,27 @@ public async Task CanWaitListenInvoice()
             }
         }
 
+        [Fact]
+        public async Task CanListChannels()
+        {
+            await EnsureConnectedToDestinations();
+            foreach (var client in Tester.GetLightningSenderClients().Concat(Tester.GetLightningDestClients()))
```

Comment:
Can you check that the channels Id of each `senders` are also present in the channels of `destinations`?